src/main/java/seedu/duke/Fitr.java

```diff
@@ -1,21 +1,43 @@
 package seedu.duke;
 
-import java.util.Scanner;
-
 public class Fitr {
-    /**
-     * Main entry-point for the java.duke.Duke application.
-     */
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("What is your name?");
+    private Storage storage;
+    private FoodList foodList;
+    private ExerciseList exerciseList;
+    private User user;
+
+    public Fitr (String filePathOfFoodList, String filePathOfExerciseList) {
```

Comment:
Remove the whitespace between `Fitr` and the open bracket, i.e. `public Fitr(...) {`